### DISCUSSION TITLE
[QUESTION] Can we import files from the same directory?

### DISCUSSION THREAD

**Discussion by rempas** [vlang/v#12019] [General]
I'm reading the docs but still can't find info on what I'm trying to do. I want to be able to spit files across the same directory. Let's consider the following structure.

```
my_awesome_project
   | src
      | main.v
      | other_file.v
   | README.md
```
So I want to import the "other_file.v" in the "main.v" file. I'm trying using things like `import other_file` or `import main.other_file` (while have defined the module first with `module main.other_file`). Any ideas?

**Comment by JalonSolov** [comment]
If they're in the same module, there is no import required.  Just compile with `v .` and V will compile all the .v files into your executable.  Basically, all files in a module look the same as if all the code was in a single file.

**Reply by rempas** [reply]
Thanks a lot! Works perfectly!
